### DISCUSSION TITLE
Watermeter don't leave Process State " Initialization"

### DISCUSSION THREAD

**Discussion by IngoSchuPe** [jomjol/AI-on-the-edge-device#3460] [Q&A]
Hi,
after update to the Version Release: v16.0.0-RC5 (Commit: 7836323+) my watermeter don't leave the Process State "Initialization". 
After several failed attempts I formated the SD-card and make a new installation. Netherless, I can no longer get out of this process state. I don't find a solution in several discussion. 

![image](https://github.com/user-attachments/assets/59c0d99f-3360-4ab1-a32f-eb13dee1e696)
![image](https://github.com/user-attachments/assets/16eda82a-9c55-4929-9593-3391816748b3)
![image](https://github.com/user-attachments/assets/c1f83f6b-99c6-432c-84b9-9d7b475bf822)

**Comment by SybexX** [comment]
![Unbenannt](https://github.com/user-attachments/assets/501b7029-e96e-48a4-beed-aa95f7e526f5)

Either a model is selected in your config.ini that is not on the SD or it was broken during copying.

![Unbenannt1](https://github.com/user-attachments/assets/091e29bd-486d-4d0e-9866-31a0604d31ec)
![Unbenannt](https://github.com/user-attachments/assets/020f7dee-bc98-45cc-9d39-bcdfb572e3b2)

**Reply by IngoSchuPe** [reply]
Hi SybexX,

thank you for your support. 

There is still one file missing in comparing with the lines that you marked. 
Could this be the reason? If I download the Version again (also the previous version) it is also not included.
Where can I get this file?

![image](https://github.com/user-attachments/assets/a5e907c3-5d7d-4f09-9227-b487cab581ca)

**Reply by SybexX** [reply]
Open the config.ini with a text editor and check whether the entered .tflite is present in the config folder, if not change it to an existing one.
The new .tflite are available in the main version https://github.com/jomjol/AI-on-the-edge-device/actions/runs/12518010749

**Reply by IngoSchuPe** [reply]
Hi SybexX,
yesterday i was busy and actual, when I look after my watermeter it works.It sounds crazy but the only think that changed is the time. Could it take such a long time for the Initializing?

**Reply by SybexX** [reply]
The page only reloads every 5 minutes and if you don't reload it manually or do nothing on the web ui, the image will remain until autorefresh.

**Comment by Ingo-Knito** [comment]
... or a webhook is active and no url set after the update.
I had this issue in the past.

The entries were set to this - not correct
```
[Webhook]
;Uri = 
;ApiKey =
;UploadImg =

```
Webhook has to lock like this - if not active
```
;[Webhook]
;Uri = 
;ApiKey = 
;UploadImg = 0

```
